recipes/flux-pmix/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flux-pmix" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flux-framework/flux-pmix/releases/download/v{{ version }}/flux-pmix-{{ version }}.tar.gz
+  sha256: f382800b1a342df0268146ea7ce33011299bf0c69a46ac8a52e87de6026c9322
+  patches:
+    - test.patch
```

Comment:
Can you add a comment on why this patch is needed?

Reply:
The tests cover the communication between multiple compute nodes. But on Azure we only have a single node available, so I disabled all the multi node tests.

Reply:
I meant add a comment in the recipe so we remember why we patched in the future. Right on top of the patch would be ok.

Reply:
That is what I did.